airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -339,7 +339,15 @@ class Charges(IncrementalStripeStreamWithUpdates):
     """
     API docs: https://stripe.com/docs/api/charges/list
     """
-    event_types = ["charge.created","charge.updated"]
+    event_types = [
```

Comment:
Is charge.created event not required or got renamed ?

Reply:
I don't see any `charge.created` evert type in the [documentation](https://docs.stripe.com/api/events/types)
We need to capture all the possible event types that can happen to a charge